Cargo.toml

```diff
@@ -29,9 +29,9 @@ rand = ">=0.5.1"
 redis = ">=0.8.0"
 regex = ">=1.0"
 reqwest = ">=0.8.5"
-rocket = ">=0.3.14"
-rocket_codegen = ">=0.3.14"
-rocket_contrib = ">=0.3.14"
+rocket = { git = "https://github.com/SergioBenitez/Rocket" }
```

Comment:
Please tag all of these to the commit, see example: https://github.com/mozilla/application-services/blob/master/Cargo.toml#L17

Reply:
Done.